process/smartContract/hooks/vmAccountsDB.go

```diff
@@ -5,45 +5,88 @@ import (
 	"math/big"
```

Comment:
Change also the fileName please: vmAAccountsDB to blockchainhookImpl

Reply:
done